### PR TITLE
*: add Lao translation

### PR DESCRIPTION
**Pull request by phatsss**
- [✅] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [✅] The page(s) have at most 8 examples.
- [✅] The page description(s) have links to documentation or a homepage.
- [✅] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [✅] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Review by phatsss — COMMENTED** [review]
i think i solved it

**Comment by kbdharun** [comment]
@navarroaxel seems like a contributor isn't present in GitHub is it fine without their CLA signature?

File: pages.la/common/piodebuggdb.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# piodebuggdb
+
+> ຄຳສັ່ງນີ້ເປັນອີກຊື່ໜຶ່ງຂອງຄຳສັ່ງ `pio debug`.
```
Comment (kbdharun) [inline pages.la/common/piodebuggdb.md:3]:
```suggestion
> ຄຳສັ່ງນີ້ເປັນອີກຊື່ໜຶ່ງຂອງຄຳສັ່ງ `pio debug --interface=gdb`.
```

File: pages.la/common/tldr.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+- ສະແດງຕົວຢ່າງການໃຊ້ຄຳສັ່ງທີ່ໃຊ້ເລື້ອຍໆ (ຄຳໃບ້: ນີ້ເປັນເຫດຜົນທີ່ທ່ານໃຊ້ບໍລິການຂອງເຮົາແມ່ນບໍ່ລ່ະ!):
+
+`tldr {{command}}`
+
+- ສະແດງໜ້າ tldr ຂອງຄຳສັ່ງ tar ສຳລັບລະບົບປະຕິບັດການ Linux:
+
+`tldr -p {{linux}} {{tar}}`
+
+- ຊໍຄວາມຊ່ວຍເຫຼືອການໃຊ້ງານຄຳສັ່ງຍ່ອຍຂອງ Git:
```
Comment (kbdharun) [inline pages.la/common/tldr.md:14]: Can you remove this as it isn't present in our main page?

File: pages.la/common/tldr.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
+# tldr
+
+> ສະແດງຕົວຢ່າງແບບງ່າຍສຳລັບເຄື່ອງມືເທິງ command-line ຈາກໂປຣເຈັກ tldr-pages.
+> ຂໍ້ມູນເພີ່ມເຕີມ: <https://tldr.sh>.
+
+- ສະແດງຕົວຢ່າງການໃຊ້ຄຳສັ່ງທີ່ໃຊ້ເລື້ອຍໆ (ຄຳໃບ້: ນີ້ເປັນເຫດຜົນທີ່ທ່ານໃຊ້ບໍລິການຂອງເຮົາແມ່ນບໍ່ລ່ະ!):
```
Comment (kbdharun) [inline pages.la/common/tldr.md:6]: I checked this description with google translate this is what it read "hint: this is why you use our service, isn't it!" shouldn't it be "hint: this is how you got here!"

Can you change this description too?

File: pages.la/common/tldr.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+`tldr {{command}}`
+
+- ສະແດງໜ້າ tldr ຂອງຄຳສັ່ງ tar ສຳລັບລະບົບປະຕິບັດການ Linux:
```
Comment (kbdharun) [inline pages.la/common/tldr.md:10]: Change this command to better align with [main page](https://github.com/tldr-pages/tldr/blob/main/pages/common/tldr.md).

File: pages.la/common/tldr.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
+# tldr
+
+> ສະແດງຕົວຢ່າງແບບງ່າຍສຳລັບເຄື່ອງມືເທິງ command-line ຈາກໂປຣເຈັກ tldr-pages.
+> ຂໍ້ມູນເພີ່ມເຕີມ: <https://tldr.sh>.
+
+- ສະແດງຕົວຢ່າງການໃຊ້ຄຳສັ່ງທີ່ໃຊ້ເລື້ອຍໆ (ຄຳໃບ້: ນີ້ເປັນເຫດຜົນທີ່ທ່ານໃຊ້ບໍລິການຂອງເຮົາແມ່ນບໍ່ລ່ະ!):
+
+`tldr {{command}}`
```
Comment (kbdharun) [inline pages.la/common/tldr.md:8]: Translate `{{command}}` and `{{subcommand}}` command arguments in this page. 

Also, add "Print the tldr page for a specific subcommand:" and it's command from our [main page](https://github.com/tldr-pages/tldr/blob/main/pages/common/tldr.md).

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I checked all translations using Google Translate. I have some suggestions for this page.

Also, if possible can you add Lao to [common-arguments](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/translation-templates/common-arguments.md), [subcommand-mention](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/translation-templates/subcommand-mention.md) and to label in [more-info-link](https://github.com/tldr-pages/tldr/blob/main/scripts/set-more-info-link.py)? 

> Thanks in advance

**Review by kbdharun — COMMENTED** [review]
Seems like Lao doesn't have a `la` language code (POSIX / ISO) instead it has `lo`(`la` is for Latin). So I will update this PR with the `lo` language code.

- References: [Wikipedia](https://en.wikipedia.org/wiki/Lao_language) , [GNU Manual](https://www.gnu.org/software/gettext/manual/html_node/Usual-Language-Codes.html)

**Comment by kbdharun** [comment]
I updated all pages except common arguments. @phatsss can you sync your branch in main so that we can add `common-arguments` too.

> Also, If possible can you fill in `subcommand-mention` too?
Thanks in advance

**Review by kbdharun — APPROVED** [review]

**Comment by kbdharun** [comment]
This PR is good to go. Thanks for your contribution.

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**